evm/test/forks/HubMainnetFork.t.sol

```diff
@@ -0,0 +1,373 @@
+  bytes32 public constant PROPOSER_ROLE = keccak256("PROPOSER_ROLE");
+  bytes32 public constant EXECUTOR_ROLE = keccak256("EXECUTOR_ROLE");
+  bytes32 public constant CANCELLER_ROLE = keccak256("CANCELLER_ROLE");
+
+  // TODO: Replace with actual proposer address with enough voting power for production verification
+  address public PROPOSER_ADDRESS = actualDeployer;
+  // TODO: Replace with actual Wormhole Foundation address for production verification
+  // address constant EXPECTED_EXTENDER_ADMIN = WORMHOLE_FOUNDATION_ADDR;
+  address public EXPECTED_EXTENDER_ADMIN = actualDeployer;
+
+  // --- Helper Functions ---
+
+  // Sets up the proposer address with delegated votes.
+  function _setupProposerAndDelegate(address _proposer) internal {
+    uint256 proposalThreshold = EXPECTED_PROPOSAL_THRESHOLD;
+    vm.prank(_proposer);
+    wToken.delegate(_proposer);
+    vm.roll(block.number + 1);
+    assertGe(wToken.getVotes(_proposer), proposalThreshold, "Proposer votes below threshold after delegation");
+  }
+
+  // Prepares data for a simple wToken.approve(gov, 0) proposal.
+  function _prepareSimpleProposalData(string memory _description)
+    internal
+    view
+    returns (address[] memory targets, uint256[] memory values, bytes[] memory calldatas, bytes32 descriptionHash)
+  {
+    targets = new address[](1);
+    targets[0] = address(wToken);
+    values = new uint256[](1);
+    values[0] = 0;
+    calldatas = new bytes[](1);
+    calldatas[0] = abi.encodeWithSignature("approve(address,uint256)", address(gov), 0);
+    descriptionHash = keccak256(bytes(_description));
+  }
+
+  // Executes gov.propose with the given parameters after pranking as _proposer.
+  function _proposeFrom(
+    address _proposer,
+    address[] memory _targets,
+    uint256[] memory _values,
+    bytes[] memory _calldatas,
+    string memory _description
+  ) internal returns (uint256 proposalId) {
+    vm.prank(_proposer);
+    proposalId = gov.propose(_targets, _values, _calldatas, _description);
+    assertTrue(proposalId != 0, "Proposal ID is zero");
+  }
+
+  // --- Setup ---
+
+  function setUp() public {
+    // Create a fork of mainnet
+    ethereumForkId = vm.createSelectFork(ETHEREUM_RPC_URL);
+
+    // Load contract instances from known addresses
+    timelock = TimelockController(payable(TIMELOCK_ADDR));
+    gov = HubGovernor(payable(GOV_ADDR));
+    extender = HubProposalExtender(EXTENDER_ADDR);
+    hubVotePool = HubVotePool(HUB_VOTE_POOL_ADDR);
+    hubProposalMetadata = HubProposalMetadata(HUB_METADATA_ADDR);
+    hubMessageDispatcher = HubMessageDispatcher(HUB_MSG_DISPATCHER_ADDR);
+    hubEvmSpokeAggregateProposer = HubEvmSpokeAggregateProposer(HUB_EVM_AGG_PROPOSER_ADDR);
+    hubSolanaMessageDispatcher = HubSolanaMessageDispatcher(HUB_SOLANA_DISPATCHER_ADDR);
+    hubSolanaSpokeVoteDecoder = HubSolanaSpokeVoteDecoder(HUB_SOLANA_VOTE_DECODER_ADDR);
+    // TODO: Replace with actual WToken address for production verification
```

Comment:
Let's remove this as we use a single variable

Reply:
https://github.com/wormhole-foundation/multigov/pull/268/commits/f53906de29e3d32f041b8cc0aeffdc02f8cb03fe